### PR TITLE
Add Symfony 2019-11 CVEs

### PR DESCRIPTION
**Pull request by jderusse**

### CODE REVIEW

File: symfony/http-foundation/CVE-2019-18888.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        versions: ['>=2.5.0', '<2.6.0']
+    2.6.x:
+        time:     ~
+        versions: ['>=2.6.0', '<2.7.0']
+    2.7.x:
+        time:     ~
+        versions: ['>=2.7.0', '<2.8.0']
+    2.8.x:
+        time:     2019-11-13 08:00:00
+        versions: ['>=2.8.0', '<2.8.51']
```
Comment (nicolas-grekas) [inline symfony/http-foundation/CVE-2019-18888.yaml]: 2.8.52

File: symfony/http-kernel/CVE-2019-18887.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        versions: ['>=2.5.0', '<2.6.0']
+    2.6.x:
+        time:     ~
+        versions: ['>=2.6.0', '<2.7.0']
+    2.7.x:
+        time:     ~
+        versions: ['>=2.7.0', '<2.8.0']
+    2.8.x:
+        time:     2019-11-13 08:00:00
+        versions: ['>=2.8.0', '<2.8.51']
```
Comment (nicolas-grekas) [inline symfony/http-kernel/CVE-2019-18887.yaml]: 2.8.52

**Comment by jderusse** [comment]
fixed `2.8.52`

File: symfony/http-foundation/CVE-2019-18888.yaml (hunk 1 above)
Comment (jderusse) [reply symfony/http-foundation/CVE-2019-18888.yaml]: good catch. I missed it

**Pull request merged**